onnx_tf/handlers/backend/q_linear_conv.py

```diff
@@ -60,6 +60,16 @@ def version_10(cls, node, **kwargs):
     new_dict[node.inputs[0]] = x
     new_dict[node.inputs[3]] = w
 
+    # if bias is defined save it here
+    B = tensor_dict[node.inputs[8]] if len(node.inputs) == 9 else tf.constant(
+        [0], tf.float32)
+    if len(node.inputs) == 9:
+      B = tf.cast(B, tf.float32)
+      B_scale = x_scale * w_scale
+      B = tf.round(B / B_scale)
+      # Remore bias from inputs
+      node.inputs.remove(node.inputs[8])
```

Comment:
Is it possible we update new_dict and leave the bias process be generally handled in conv_mixin?